packages/gatsby-source-drupal/src/gatsby-node.js

```diff
@@ -29,7 +32,7 @@ exports.sourceNodes = async (
   // Default concurrentFileRequests to `20`
   concurrentFileRequests = concurrentFileRequests || 20
 
-  // Touch existing Drupal nodes so Gatsby doesn't garbage collect them.
+  // Touch existing Drupal nodes so Gatsby doesn`t garbage collect them.
```

Comment:
this update seems like a mistake?

Reply:
yes, the result of moving from a different prettier and lint configuration (a find and replace) ill revert it